service/src/main/resources/config/verily/sdd_refresh0323/original/ui/top_level.json

```diff
@@ -265,7 +265,7 @@
       "id": "tanagra-snp",
       "title": "SNP variant",
       "category": "Program data",
-      "columns": [{ "key": "id", "width": "100%", "title": "SNP variant" }],
```

Comment:
@tjennison-google Here's the change I made to the UI config so that the new `name` attribute (previously the `id` attribute) is displayed in the list. PTAL.